### PR TITLE
PR of jkotalik PR on HTTP logging

### LINKED ISSUE #22159: Add HttpLogging Middleware documentation

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/http-logging/?view=aspnetcore-6.0&branch=pr-en-us-22103)

Fixes  #22159

### CODE REVIEW

File: aspnetcore/fundamentals/http-logging/index.md [hunk 1]
```diff
@@ -12,88 +12,95 @@ uid: fundamentals/http-logging/index
...
-ResponseHeaders are a set of HTTP Request Headers that are allowed to be logged. Header values will only be logged for header names that are in this collection.
+`ResponseHeaders` are a set of HTTP Response Headers that are allowed to be logged. Header values are only logged for header names that are in this collection.
 
 [!code-csharp[](samples/6.x/Startup.cs?name=configureservices&highlight=8)]
 
-### MediaTypeOptions
+### `MediaTypeOptions`
 
-MediaTypeOptions provides configuration for selecting which encoding to use for a specific media type. 
+`MediaTypeOptions` provides configuration for selecting which encoding to use for a specific media type.
```
Comment (jkotalik) [inline aspnetcore/fundamentals/http-logging/index.md:92]: I think I forgot to expand on this type: https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/HttpLogging/src/MediaTypeOptions.cs. Could you follow a similar model to what we did prior with options?

**Review by jkotalik — APPROVED** [review]
LGTM pending that change.

File: aspnetcore/fundamentals/http-logging/index.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/fundamentals/http-logging/index.md:92]: @serpent5 can you point out what options need documenting?

Comment (jkotalik) [reply aspnetcore/fundamentals/http-logging/index.md:92]: The entire class need documentation (isn't a lot)
AddText and overloads
AddBinary and overloads
Clear

**Pull request merged**